test/remote/gateways/remote_adyen_test.rb

```diff
@@ -993,6 +993,14 @@ def test_successful_tokenize_only_store
     assert_equal 'Success', response.message
   end
 
+  def test_successful_tokenize_only_store_with_ntid
```

Comment:
I'm not sure if there are additional assertions that would make sense to add here...from what I could tell, it doesn't seem like Adyen's response really changes with the addition of this field to the request 🤔

Reply:
This looks good, I wonder if you were to add the ntid to a different transaction type does it change those responses vs when it's not there? I don't know that that's relevant in terms of THIS test, but would maybe explain why it's not changing in the store response. Not asking you to go back and investigate that, just came to mind.